__tests__/app.test.js

```diff
@@ -190,3 +191,45 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id ", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.comments).toEqual([
+          {
+            comment_id: 14,
+            votes: 16,
+            created_at: "2020-06-09T05:00:00.000Z",
+            author: "icellusedkars",
+            body: "What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.",
+          },
+          {
+            comment_id: 15,
+            votes: 1,
+            created_at: "2020-11-24T00:08:00.000Z",
+            author: "butter_bridge",
+            body: "I am 100% sure that we're not completely sure.",
+          },
+        ]);
+      });
+  });
```

Comment:
add another expect to check if the length of the array is what we expect it to be